runtime/src/execute_cost_table.rs

```diff
@@ -45,19 +47,25 @@ impl ExecuteCostTable {
         self.table.len()
     }
 
-    // instead of assigning unknown program with a configured/hard-coded cost
-    // use average or mode function to make a educated guess.
-    pub fn get_average(&self) -> u64 {
+    // default program cost to max_units
+    pub fn get_default_units(&self) -> u64 {
+        let use_max_units_default = false;
+        ComputeBudget::get_max_units(use_max_units_default)
```

Comment:
This is unintuitive to read because `get_max_units()` and `get_default_units()` seem unrelated, yet one calls the other.

I think if we follow this suggestion here: https://github.com/solana-labs/solana/pull/23656/files#r827652688, we can just directly use the constants instead of making function calls to indirectly get the constants

Reply:
+1